src/status_im/ui/screens/wallet/main/styles.cljs

```diff
@@ -26,9 +26,7 @@
   {:background-color styles/color-blue4})
```

Comment:
if we have colors, why still use styles ?

Reply:
i mean in this ns, we have colors

Reply:
:+1:

Reply:
I plan to do this PR by PR